twiml/voice/pay/prompt/full-example-credit-card/full-example-credit-card.3.x.js

```diff
@@ -0,0 +1,89 @@
+const VoiceResponse = require('twilio').twiml.VoiceResponse;
+
+const response = new VoiceResponse();
+const pay = response.pay({
+    paymentMethod: 'credit-card',
+    validCardTypes: 'visa mastercard amex'
+});
+const prompt = pay.prompt({
+    for: 'payment-card-number'
+});
+prompt.say('Please enter your credit card number.');
+const prompt2 = pay.prompt({
+    for: 'payment-card-number',
+    errorType: 'timeout'
+});
+prompt2.say('I am sorry I didn\'t hear anything. Please enter your credit card number.');
```

Comment:
nit: I think there should be a comma after "I am sorry", or "I am sorry" can be removed. (note that this should be a find/replace to catch other instances if you decide to modify this.

Reply:
Good point!